__tests__/app.test.js

```diff
@@ -362,3 +362,36 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: No Content - Comment deleted", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then((body) => {
+        expect(body.msg).toBe(undefined);
```

Comment:
It is strange on a delete/204, but yeah you don't get anything back. You don't need to check for a body.msg!